addons/sourcemod/scripting/szf/weapons.sp

```diff
@@ -125,19 +132,31 @@ void Weapons_ReplaceEntityModel(int ent, int index)
 		
 		if (index == wep.iIndex)
 		{
-			SetWeaponModel(ent, wep.sModel);
+			SetWeaponModel(ent, wep);
 			return;
 		}
 	}
 }
 
+// enum structs are truly epic
+void Weapons_GetOffsets(eWeapon wep, float origin[3], float angles[3])
```

Comment:
I'm really unsure whats the problem when you can just directly grab from struct, similar to what i mentioned when loading config

Reply:
Same `array sizes do not match, or destination array` error whenever i try to pass an array from enum struct